pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +124,18 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if errors.IsNotFound(err) {
+					runningStrategy, invalid := vm.RunStrategy()
```

Comment:
please use `err` instead of `invalid`, it's best practice

Reply:
thought I was using both... my bad